### PR TITLE
App ray mobile security 1.3.9

### PR DESCRIPTION
**Pull request by kaldyka**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/app-ray-mobile-security/1.3.9/step.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    description: |-
+      Your App-Ray base URL
+      Example https://demo.app-ray.co/
+      URL MUST end with /
+    is_required: true
+    summary: "Your App-Ray base URL \nExample https://demo.app-ray.co/"
+    title: AppRay instance base URL
+- app_path: null
+  opts:
+    description: Path to the app file you want to upload, generated by Bitrise, usually
```
Comment (koral--) [inline steps/app-ray-mobile-security/1.3.9/step.yml:42]: Is AAB supported?
I think it is worth confirming or denying.

**Review by zsmav — APPROVED** [review]

**Review by zsmav — APPROVED** [review]

**Review by zsmav — APPROVED** [review]

File: steps/app-ray-mobile-security/1.3.9/step.yml (hunk 1 above)
Comment (zsmav) [reply steps/app-ray-mobile-security/1.3.9/step.yml:42]: It is supported

**Review by zsmav — APPROVED** [review]

File: steps/app-ray-mobile-security/1.3.9/step.yml (hunk 1 above)
Comment (kaldyka) [reply steps/app-ray-mobile-security/1.3.9/step.yml:42]: Added it to title and escription

**Review by trapacska — APPROVED** [review]

**Comment by trapacska** [comment]
Thanks for the Step @kaldyka ! 🚀

**Comment by Janos-vrancsik** [comment]
@kaldyka thanks for the contribution!

We're always striving to improve the development experience. 

Could you please help us improve further by filling out [this short survey](https://www.surveygizmo.eu/s3/90265612/Verified-Step-Development)?

Thanks in advance!

**Pull request merged**